chef_master/source/config_rb_policyfile.rst

```diff
@@ -49,12 +49,14 @@ A Policyfile.rb file may contain the following settings:
    Required. The run-list the chef-client will use to apply the policy to one (or more) nodes.
 
 ``default_source :SOURCE_TYPE, *args``
-   The location in which any cookbooks not specified by ``cookbook`` are located. Possible values: ``chef_repo``, ``:community``, and ``:supermarket``. Use more than one ``default_source`` to specify more than one location for cookbooks.
+   The location in which any cookbooks not specified by ``cookbook`` are located. Possible values: ``chef_repo``, ``chef_server``, ``:community``, and ``:supermarket``. Use more than one ``default_source`` to specify more than one location for cookbooks.
```

Comment:
So, @tduffield just gave me a full lesson about this so I'm ok with the current state.. 😂 Thanks Tom! 🎉